src/components/Tooltip/__stories__/tooltip.stories.mdx

```diff
@@ -28,47 +26,64 @@ export const tooltipTemplate = (args) => {
         <div />
       </Tooltip>
     </div>
-  )
-}
+  );
+};
 
 <!--- Component documentation -->
 
 # Tooltip
+
 - [Overview](#overview)
 - [Props](#props)
 - [Usage](#usage)
 - [Variants](#variants)
+- [Themes](#themes)
 - [Do’s and don’ts](#dos-and-donts)
 - [Use cases and examples](#use-cases-and-examples)
 - [Related components](#related-components)
 - [Feedback](#feedback)
 
 ## Overview
+
 Tooltips show contextual help or information about specific components when a user hovers on them.
+
 <Canvas>
-  <Story name="Overview"
-         args={ {shouldShowOnMount: true, content: "I’m a tooltip"} }>
-    { tooltipTemplate.bind({}) }
+  <Story name="Overview" args={{ shouldShowOnMount: true, content: "I’m a tooltip" }}>
+    {tooltipTemplate.bind({})}
   </Story>
 </Canvas>
 
 ## Props
-<ArgsTable of={ Tooltip } />
+
+<ArgsTable of={Tooltip} />
 
 ## Usage
-<UsageGuidelines guidelines={[
-  "Should never contain critical information that a user would absolutely need to proceed.",
-  "Use when the description content would be too much information to include inline or create too much clutter. For example, when expert users have seen it many times.",
-]}/>
-
-<Tip>As tooltips only surface from a hover, never include links or buttons in the copy. If your tooltip requires either of these, considers putting your content in a
-  <Link href="/?path=/docs/popover-tipseen--overview" size={Link.sizes.SMALL} withoutSpacing>Tipseen</Link> or
-   <Link href="/?path=/docs/feedback-attentionbox--overview" size={Link.sizes.SMALL}>Attention box.</Link>
+
+<UsageGuidelines
+  guidelines={[
+    "Should never contain critical information that a user would absolutely need to proceed.",
+    "Use when the description content would be too much information to include inline or create too much clutter. For example, when expert users have seen it many times."
+  ]}
+/>
+
+<Tip>
+  As tooltips only surface from a hover, never include links or buttons in the copy. If your tooltip requires either of
+  these, considers putting your content in a
+  <Link href="/?path=/docs/popover-tipseen--overview" size={Link.sizes.SMALL} withoutSpacing>
+    Tipseen
+  </Link>{" "}
```

Comment:
is this addition (`{" "}`) intentional?

Reply:
If the `{""}` are not intentional let's remove them :)